rules/the-best-way-to-manage-assets-in-xamarin/rule.md

```diff
@@ -3,7 +3,7 @@ type: rule
 archivedreason: 
 title: Do you know the best way to manage assets in Xamarin?
 guid: eb298f24-ec4e-492f-ab66-3de736a6ee3d
-uri: the-best-way-to-manage-assets-in-xamarin
+uri: the-best-way-to-manage-assets-in-maui
```

Comment:
The categories need to have the uri changed too, and the folder name needs to be changed

Reply:
Decided to archive the rule.